struqture-py/docs/conf.py

```diff
@@ -3,11 +3,11 @@
 #
 """Configuration of sphinx documentation module"""
 try:
-    with open('../struqture_py/__version__.py') as f:
+    with open("../struqture_py/__version__.py") as f:
         lines = f.readlines()
     version = lines[-1].strip().split("'")[1].strip()
 except Exception:
-    version = '0.0.0'
+    version = "0.0.0"
 versions = version.split(".")
 main_version = "{}.{}".format(versions[0], versions[1])
```

Comment:
```py
import tomli

main_version = tomli.load(open("../pyproject.toml", "rb"))["project"]["version"]
```
But requires `tomli` of course. I see that we are not using the normal workflow for the docs, on qoqo we would put it under `[project.optional-dependencies]` in the `pyprojec.toml`, I guess here we have to do it manually by modifying line 23 of the manual_user_documentation yml.